src/main/webapp/app/admin/upcoming-exams-and-exercises/upcoming-exams-and-exercises.component.ts

```diff
@@ -24,11 +24,11 @@ export class UpcomingExamsAndExercisesComponent implements OnInit {
 
     ngOnInit(): void {
         this.exerciseService.getUpcomingExercises().subscribe((res: ExerciseEntityArrayResponseType) => {
-            this.upcomingExercises = res.body ?? [];
+            this.upcomingExercises = res.body === null ? [] : res.body;
```

Comment:
```suggestion
            this.upcomingExercises = res.body == undefined ? [] : res.body;
```
Why are you checking specifically for `null`? We want to avoid using `null` use `undefined` instead!
`== undefined` will check for both `undefined` and `null`, but please use undefined!

Reply:
Why did you change the code in the first place? `??` checks for `null` or `undefined` see: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

Reply:
Thanks for the feedback @fde312! 
The reason behind that is that `res.body` has the `HttpResponse<T>.body: Exam[] | null` type. I just checked the other code and it seems I can just do a check like this: ` res.body ?? []`.

Reply:
@ivanchimeno Please use type `Exam[] | undefined` instead

Reply:
@fde312 I don't think I can change that because `res.body` is not coming from me. `res` is of type `HttpResponse<Exam[]>` and `HttpResponse<Exercise[]>`. `res.body` is a property of `HttpResponse` and has the type `readonly body: T | null;`